### PR TITLE
Exposed volume prop in HowlOptions

### LINKED ISSUE #17: Support volume property
I would like to propose exposing
https://github.com/goldfire/howler.js#volume-number-10 
in
src/Howler.Blazor/Components/HowlOptions.cs 
as double? in order to have better control on how multiple sounds are played.

### PR DESCRIPTION
**Pull request by k-u-s**
Resolves #17,

Added Volume property to HowlOptions and added Play half and 1/4 buttons to sample apps.

### CODE REVIEW

File: examples/Howler.Blazor-AudioPlayer/Pages/Example.razor [hunk 1]
```diff
@@ -19,6 +19,14 @@
             <td><button class="btn btn-primary oi oi-media-play" @onclick="Play3"></button></td>
             <td>Play an audio file with source location and format</td>
         </tr>
+        <tr>
+            <td><button class="btn btn-primary oi oi-media-play" @onclick="PlayHalf"></button></td>
```
Comment (StefH) [inline examples/Howler.Blazor-AudioPlayer/Pages/Example.razor:23]: Can you rename `PlayHalf` to `PlayAudioWithHalfVolume` in all code?

File: examples/Howler.Blazor-AudioPlayer/Pages/Example.razor [hunk 2]
```diff
@@ -19,6 +19,14 @@
             <td><button class="btn btn-primary oi oi-media-play" @onclick="Play3"></button></td>
             <td>Play an audio file with source location and format</td>
         </tr>
+        <tr>
+            <td><button class="btn btn-primary oi oi-media-play" @onclick="PlayHalf"></button></td>
+            <td>Play an audio file with source location, format and volume set to half</td>
+        </tr>
+        <tr>
+            <td><button class="btn btn-primary oi oi-media-play" @onclick="PlayQuoter"></button></td>
```
Comment (StefH) [inline examples/Howler.Blazor-AudioPlayer/Pages/Example.razor:27]: Can you rename `PlayQuoter` to `PlayAudioWithQuarterVolume` in all code?

**Review by StefH — CHANGES_REQUESTED** [review]
Please take a look at my comments

File: examples/Howler.Blazor-AudioPlayer/Pages/Example.razor (hunk 2 above)
Comment (k-u-s) [reply examples/Howler.Blazor-AudioPlayer/Pages/Example.razor]: Done in https://github.com/StefH/Howler.Blazor/pull/18/commits/f609ccb4cf03709844d67fca14b29a1a00a91662

File: examples/Howler.Blazor-AudioPlayer/Pages/Example.razor (hunk 1 above)
Comment (k-u-s) [reply examples/Howler.Blazor-AudioPlayer/Pages/Example.razor]: Done in https://github.com/StefH/Howler.Blazor/pull/18/commits/f609ccb4cf03709844d67fca14b29a1a00a91662

**Review by StefH — APPROVED** [review]

**Pull request merged**